hook.js

```diff
@@ -128,6 +128,9 @@ ${exportNames.map((n) => `
 let $${n} = namespace.${n}
 export { $${n} as ${n} }
 set.${n} = (v) => {
+  if ('${n}' !== 'default' && namespace.default['${n}']) {
+    namespace.default['${n}'] = v
```

Comment:
I think you need to make sure that this only applies to CJS modules. This could easily be incorrect for ESM modules.

Reply:
Noted. I'm putting this PR on hold until we have resolution on my others. Even if this one were merged, I'd still have to proxy the returned proxy because I need a symbol to detect the proxy. So I want to write up a case for that and discuss it.